flujo/recipes/factories.py

```
@@ -411,17 +411,60 @@
+                    checklist = step.output
+                    break
+            except Exception:
+                continue
+
+    # 3) If still missing, fallback to the review scratchpad checklist
+    if checklist is None:
+        try:
+            if context is not None and hasattr(context, "scratchpad"):
+                checklist = context.scratchpad.get("checklist")
+        except Exception:
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)